knowledge/science/botony/poison_ivy/qna.yaml

```diff
@@ -0,0 +1,19 @@
+created_by: Jeff Pacheco
+domain: medical
+seed_examples:
+- answer: | Toxicodendron orientale: found in East Asia. Toxicodendron radicans: found throughout eastern Canada and the United States, Mexico and Central America, Bermuda and the Bahamas. Toxicodendron rydbergii: found throughout Canada and much of the United States except the southeast.
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!